commissioning.tex

```diff
@@ -1,17 +1,25 @@
 \section{Rubin Observatory Commissioning}
 \label{sec:commissioning}
 
-The baseline schedule for on-sky observations during SIT-COM includes six months of technical integration and testing, and concludes with an 8-week period of sustained observing in the form of two \svs \citep{SCTN-007}.
+The baseline schedule for on-sky observations during SIT-COM includes six months of technical integration and testing, and concludes with an 8-week period of sustained observing in the form of two LSST Cam \svs \citep{SCTN-007}.
 
-\subsection{Commissioning Observations}
+\subsection{Commissioning observations}
 
-\TODO{Phil, for review and extension as needed, by Keith}{New subsection with high level summary of SIT-COM's observing plans, following placeholder notes below}.
+\begin{figure}
+\includegraphics[width=\linewidth]{figures/commissioning-plan}
+\caption{Outline plan for the collection of commissioning data, as of October 2022.}
+\label{fig:commissioning}
+\end{figure}
 
-Figure from Keith's slides showing Engineering, System Optimization, Science Validation phases.
+Figure~\ref{fig:commissioning} shows the high level plan for the Rubin commissioning observations, first with the $40\times40$~arcmin field of view Commissioning Camera ``ComCam,'' and then with the LSST Camera.
+Commissioning data collection is planned to take place in phases.
+A period of on-sky engineering time will be followed by a set of observations designed to help optimize the system, before the \svs are carried out.
+The SV surveys are designed to support scientific analyses that validate the system's performance, and allow Rubin to demonstrate operations readiness.
 
-Text introducing figure.
-
-Key components of SO and SV phases with each Camera. 20 year depth tests. 1000 sq deg pilot survey.
+The figure indicates a number of key components of the system optimization and SV phases planned for each camera.
+These include 20 year depth tests in select fields, and a 1000 sq.\ deg.\, multi-band, wide-fast-deep (WFD) 1-2 year depth ``pilot'' survey.
+Field selection will be carried out by the Commissioning Team, taking into account a wide variety of constraints as well as a ``menu'' of science opportunities provided by the LSST Science Community.
```

Comment:
The community alone will not decide the menu. I'd rephrase as 

Field selection will be carried out by the Commissioning Team, taking into account a wide variety of constraints  as well as a ``menu'' of science opportunities to which the LSST Science Community may contribute.

Reply:
Thanks, that's better. The community has already contributed to the menu, so I'll make sure that comes across.